build.gradle

```diff
@@ -67,10 +67,6 @@ dependencies {
     compile(postgresClient) {
         exclude group: 'org.slf4j'
     }
-    compile ("io.dropwizard:dropwizard-lifecycle"){
-        exclude group: 'com.google.guava', module: 'guava'
```

Comment:
I'm unfamiliar with gradle, does removing `compile "com.google.guava:guava:27.0-jre"` mean that it will no longer be included in the final artefact/jar? Could you please help me understand?

Reply:
Guava is already included in the jar, this was just overriding the old version previously included by dropwizard i.e. this is the pattern described in https://stackoverflow.com/a/30729087/238230

Reply:
ah ok that makes sense, I can see how its working now :)